ppr-ui/src/components/mhrRegistration/YourHome/OtherInformation.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <v-card flat class="white pb-6 pt-6 pr-10 pl-8 rounded">
+      <v-row>
+        <v-col cols="2">
+          <label class="generic-label ml" for="">Other</label>
+        </v-col>
+        <v-col cols="10">
+          <v-textarea
+            v-model="otherRemarks"
+            filled
+            :error-messages="errorMessages.otherRemarks"
```

Comment:
This is a manual approach to applying an error message, see comments below about utilizing `:rules` prop perhaps.

Reply:
Yes, will rework it.